tests/UI/campaigns/functional/WS/01_countriesCRUD.ts

```diff
@@ -0,0 +1,611 @@
+              createDocument,
+            ) as string;
+            const createNodeValueFR: string = xpath.select1(
+              `string(/prestashop/country/${objectNode.nodeName}/language[@id="2"])`,
+              xmlDocument,
+            ) as string;
+            expect(objectNodeValueFR).to.be.eq(createNodeValueFR);
+          } else {
+            const objectNodeValue: string = xpath.select1(
+              `string(/prestashop/country/${objectNode.nodeName})`,
+              createDocument,
+            ) as string;
+            expect(objectNode.textContent).to.be.eq(objectNodeValue);
+          }
+        }
+      });
+
+      it('should go to \'International > Locations\' page', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'goToLocationsPage', baseContext);
+
+        await dashboardPage.goToSubMenu(
+          page,
+          dashboardPage.internationalParentLink,
+          dashboardPage.locationsLink,
+        );
+        await zonesPage.closeSfToolBar(page);
+
+        const pageTitle = await zonesPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(zonesPage.pageTitle);
+      });
+
+      it('should go to \'Countries\' page', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'goToCountriesPage', baseContext);
+
+        await zonesPage.goToSubTabCountries(page);
+
+        const pageTitle = await countriesPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(countriesPage.pageTitle);
+      });
+
+      it('should filter country by ID', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'filterToUpdate', baseContext);
+
+        // Filter
+        await countriesPage.resetFilter(page);
+        await countriesPage.filterTable(page, 'input', 'id_country', countryNodeID);
+
+        // Check number of countries
+        const numberOfCountriesAfterFilter = await countriesPage.getNumberOfElementInGrid(page);
+        await expect(numberOfCountriesAfterFilter).to.be.eq(1);
+
+        const textColumn = await countriesPage.getTextColumnFromTable(page, 1, 'id_country');
+        await expect(textColumn).to.contains(countryNodeID);
+      });
+
+      it('should go to edit country page', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'goToEditCountryPage', baseContext);
+
+        await countriesPage.goToEditCountryPage(page, 1);
+
+        const pageTitle = await addCountryPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(addCountryPage.pageTitleEdit);
+      });
+
+      it('should check all values', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'checkEditCountryValues', baseContext);
+
+        const createDocument: Document = domParser.parseFromString(xmlCreate);
+
+        const xmlValueIDZone = xpath.select1('string(/prestashop/country/id_zone)', createDocument) as string;
```

Comment:
Maybe it's better to create a helper for ws methods, wdyth ?

Reply:
Fixed